docs/usage/self-hosted-configuration.md

```diff
@@ -895,6 +895,17 @@ It also may mean that ignored directories like `node_modules` can be preserved a
 
 ## platform
 
+## platformVersion
+
+Specify this string for Renovate to skip API checks and provide GitLab/Bitbucket server version directly.
+Particularly useful with GitLab's `CI_JOB_TOKEN` to authenticate Renovate or to reduce API calls.
```

Comment:
```suggestion
When you set a `platformVersion` string, Renovate skips a API call to GitLab/Bitbucket.
Instead of fetching the version from the API, Renovate uses the version in the `platformVersion` string.

Setting a `platformVersion` manually is handy when you:

- use GitLab's `CI_JOB_TOKEN` to authenticate Renovate, or
- want to reduce the number of API calls
```